### PR TITLE
vendor: pin etcd to v3.1.8

### PR DESCRIPTION
**Pull request by hongchaodeng**
Just ran `make vendor` on bootkube master and have this change.

### CODE REVIEW

**Comment by hongchaodeng** [comment]
Seems like etcd package isn't versioned in glide.yaml .
Pin it.

**Comment by dghubble** [comment]
LGTM. Can this bump to v3.2.0 or not yet?

**Comment by hongchaodeng** [comment]
Per discussion in slack, we pin versions to whatever is in glide.lock right now to start with.

File: glide.yaml [hunk 1]
```diff
@@ -5,11 +5,14 @@ import:
   subpackages:
   - pkg/spec
 - package: github.com/coreos/etcd
+  version: a73950545a5d77e02ba29432ec07d2a0eba5e830
```
Comment (diegs) [inline glide.yaml]: Can you bump this one to an actual release tag?

**Comment by hongchaodeng** [comment]
@diegs 
Updated etcd dep to v3.1.8

**Comment by hongchaodeng** [comment]
coreosbot run e2e

**Comment by hongchaodeng** [comment]
coreosbot run e2e

**Comment by hongchaodeng** [comment]
coreosbot run e2e

**Comment by hongchaodeng** [comment]
coreosbot run e2e

**Pull request merged**